lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -86,9 +98,26 @@ def uri
           @uri = graph_service.subjects_for_object_value(graph: @filtered_graph, predicate: RDF::URI.new(term_config.term_results_id_predicate), object_value: id.gsub('%20', ' ')).first
         end
 
+        def ldpaths_for_term
+          label_ldpath = term_config.term_results_label_ldpath
+          return {} if label_ldpath.blank?
+          ldpaths = { label: label_ldpath }
+          ldpaths.merge(optional_ldpaths)
+        end
+
+        def optional_ldpaths
+          opt_ldpaths = {}
+          opt_ldpaths[:altlabel] = term_config.term_results_altlabel_ldpath
+          opt_ldpaths[:id] = term_config.term_results_id_ldpath
+          opt_ldpaths[:narrower] = term_config.term_results_narrower_ldpath
+          opt_ldpaths[:broader] = term_config.term_results_broader_ldpath
+          opt_ldpaths[:sameas] = term_config.term_results_sameas_ldpath
+          opt_ldpaths.delete_if { |_k, v| v.blank? }
+        end
+
         def preds_for_term
           label_pred_uri = term_config.term_results_label_predicate
-          raise Qa::InvalidConfiguration, "required label_predicate is missing in configuration for LOD authority #{authority_name}" if label_pred_uri.nil?
```

Comment:
Since the requirement now is that there be **either** predicate map OR ldpath map, the check and raising of an exception is moved up to the caller of this method.